open-api/rest-catalog-open-api.yaml

```diff
@@ -3041,6 +3096,34 @@ components:
         - $ref: '#/components/schemas/AddViewVersionUpdate'
         - $ref: '#/components/schemas/SetCurrentViewVersionUpdate'
 
+    OperationType:
+      type: string
+      description: >
+        Defines the type of operation, either a standard operation defined by the Iceberg REST API
+        specification or a custom operation specific to an implementation.
+      anyOf:
+      - enum:
+        - create-table
```

Comment:
what about just using create/update/drop/rename as the operation type and then have an entity/resource type, which can be a table/view/namespace/X

Reply:
I think we agreed in the wider community in the sync on the current approach.
If you feel this needs more discussion, please re-open :)